stubs/ujson/ujson.pyi

```diff
@@ -10,6 +10,10 @@ def encode(
     escape_forward_slashes: bool = ...,
     sort_keys: bool = ...,
     indent: int = ...,
+    allow_nan: bool = ...,
+    reject_bytes: bool = ...,
+    default: Any = ...,
+    separators: Any = ...,
```

Comment:
```suggestion
    default: Callable[[Unknown], Unknown] | None = None,
    separators: tuple[str, str] | None = None,
```

Same for the other two instances below. Needs import of `_typeshed.Unknown` and possible `Callable`.

Reply:
Yikes, indeed.

Reply:
thanks for the feedback! have improved the precision as suggested